Test cs read-write. Log scenario

Emgariko opened the pull request:
### Changelog entry <!-- a user-readable short description of the changes that goes to CHANGELOG.md and Release Notes -->

Test https://github.com/ydb-platform/ydb/issues/13530

### Changelog category <!-- remove all except one -->
* Not for changelog (changelog entry is not required)

### Description for reviewers <!-- (optional) description for those who read this PR -->

...

Commit:
init test

Commit:
return

nemo-cpt committed:
List all cython gen-time dependencies
commit_hash:5d61d277db787b6b0c1ecf18add8ba761ff8b39f

Nechda committed:
Dont sanitize OpenSSL init function
commit_hash:a0afc0a8138dbe50df27d0096f8e500232d9dbc3

Commit:
Cosmetics
commit_hash:cc05ee0af7c8dc14d19fcbdbc3fb1cf376c82ddb

Commit:
[java] YT-20444 Move YTsaurusFixture to test-integration

* Changelog entry
Type: fix
Component: java-sdk

Move YTsaurusFixture from ytsaurus-testlib to ytsaurus-client/src/test-integration.
commit_hash:1a6d6bf9b5bb37e63cd50efb67acae0dd1e27e55

dorooleg commented on ydb/tests/olap/test_log_scenario.py:
```diff
@@ -0,0 +1,160 @@
+        logging.info(f'YdbWorkloadLog {operation_name}')
+        command = self.begin_command + [
+            "run",
+            str(operation_name),
```
Добавь пожалуйста сюда параметр `--timestamp_deviation 180`

Emgariko replied:
DONE

igormunkin committed:
YQL-19700: Make DateTime UDF overloads backward compatible

Follows up e9d589d43101dde8ca86955627fca9dbdaba1f7b
Follows up ed323f55ce6ca64b9a912772866d7bfb4fc1235f
Follows up 3b203e7a313e54ee57069763da5cf1b59991d66c
Follows up 99b69544bd45b52e2e6bfcd8cd51e34827a0f08d
Follows up 3afbcdd00ad47ecf35cb29bb25a5d7658ed1d5cb
Follows up ca9883df39a8c08e7f0d83bafcbb8853f243ed04
Follows up cfaa3532a6554646b11f204e6f4cc7677d428b8b
Follows up 9dcc43f4facdab5b4fca0a4079833c8534359124
commit_hash:7b8d84f03c1c623e6d076da42c5f0c3437f7acf6

alexv-smirnov committed:
Import libraries 250314-1301

dimdim1177 committed:
Revert commit rXXXXXX,Fix api/implementation in dependencies
commit_hash:339fba10db40dbbc519f4c287592c09239966793

Emgariko committed:
refactoring

Emgariko commented on ydb/tests/olap/test_log_scenario.py:
```diff
@@ -0,0 +1,160 @@
+            hours: int = random.randint(1, 10)
+            self.ydb_client.query(f"SELECT COUNT(*) FROM `{self.table_name}` ")
+            self.ydb_client.query(f"SELECT * FROM `{self.table_name}` WHERE ts < CurrentUtcTimestamp() - DateTime::IntervalFromHours({hours})")
+            self.ydb_client.query(f"SELECT COUNT(*) FROM `{self.table_name}` WHERE ts < CurrentUtcTimestamp() - DateTime::IntervalFromHours({hours})")
```
DONE

Emgariko commented:
# ADD deviation param
# Exception check - и выдавать нормальное сообщение об ошибке
# Добавить запросы часовые 
# Убрать параметр insert_mode, делать сразу все
# переписать на BaseTestSet
# Перенести в olap/scenario

vitstn committed:
YQL-19694 ABI logger interface
commit_hash:b673d403e9bfb2ac31eedd8d2231ac018a8f36dd

Emgariko commented on ydb/tests/olap/helpers/ya.make:
```diff
@@ -0,0 +1,7 @@
+PY3_LIBRARY()
```
DONE

rvu1024 committed:
Proper table snapshot usage in hybrid and file gateway

Суть проблемы и фикса: При сохранении анонимной таблицы в таблицу снепшотов в качестве ключа используется полный путь таблицы (после TransformPath). В методе GetRealTable, который используется в DQ и hybrid, в таблице снепшотов искали по исходному имени таблицы, до резолвига. И если не находили, то использовали текущий путь таблицы. Для анонимных таблицы получалось, что всегда читали текущее состояние таблицы, а не из снепшота. Аналогичная проблема была в методах DownloadTables (используется только в mixed gateway) и GetTablePartitions (мог аффектить DQ в режиме force). Поправил поиск по ключу с уже отрезолвленным именем, сделал безусловный поиск снепшотов для не временных и анонимных таблиц и поставил ENSURE на наличие снепшота
commit_hash:0a19e33a7de3196cce1639bae7ed17c7042e028c

robot-piglet committed:
Intermediate changes
commit_hash:e4d74265cffda90e1a7a702fed4b2e0dc1225a8c

Emgariko committed:
refactoring

Commit:
YT-23989: Move memory tracker to library/cpp/yt
commit_hash:5dad6ada81567dcb5da6ef1efe47e1738196d219

Commit:
YT-24018: Add experiment with support for NBD chunk disks.
commit_hash:7d12518000e23c9c46c60fbd14d51dd863bbbf5b

Commit:
Revert cli@4.56.0
commit_hash:b0eb7464de22bea04208d2ab423c4b4c38e385c9

Commit:
YT-24477: Introduce CannotRemoveNonemptyCompositeNode error code
commit_hash:2314e6a9da110652ef2c71a41cdb0b925df9e0c4

Commit:
YT-20778: Make job_count configurable for vanillas

* Changelog entry
Type: feature
Component: map-reduce

Allow to change job_count on-the-fly for vanilla tasks
commit_hash:821a68d7f8bbd9373dc8986529f2cb1c86925a92

Commit:
Change "build/platform/yfm/resources.json"
commit_hash:22de32d180de7e583a1427b935968849a9f2dbe4

andrey-khropov committed:
Do not add Cython standard Includes directory to Cython include path explicitly in target_cython_include_directories. Needed for compatibility with NumPy 2.x.. #2671.. #2810
commit_hash:cb90858abce76a2dc3bc3e21717522484274c39f

Commit:
Drop legacy replicas
commit_hash:4f9a33df0ac755791f57d0c4f56339b6239d30e1

Commit:
Update YFM for ya tool to latest version

Update YFM for ya tool to latest version
commit_hash:0643950859b76682ca1d893e1dc1bf6f2d5aee72

Commit:
PR from branch users/3y3k0/revert

Revert "Update cli version to 4.52.1"
This reverts commit e1018f8899ba6c3930b88c0e57c9cd333cfcd9ec, reversing
changes made to 0da0bce1d14e6c6e22b67b2fe42122931c35b840.

Revert "Update cli version to 4.53.1"
This reverts commit cdf9b2a2ea6f9b9a8e7febcaca4f933587b86bd3, reversing
changes made to 92d5f0e6ed31f5ed31f70196137e5ef02751e720.

Revert "Update cli version to 4.53.3"
This reverts commit 837b5998010cacfa37f9be5fc661c01891d1fe00, reversing
changes made to a9f3ca00638d91b200c81cbb0bda2df49fb91296.

Revert "Update cli version to 4.53.4"
This reverts commit c665f9fae736e004416286e532a762513fcb0cce, reversing
changes made to aefc0de1f5143d0d7b8478486178863251ab24ee.

Revert "Update cli version to 4.53.5"
This reverts commit d247712a1ebc5887400903662812eb248ceeb648, reversing
changes made to 87ddb5201bf0eb911ca3ede81a45c21e698c59b9.

Revert "Update cli version to 4.53.7"
This reverts commit 0358b3b2857a9197370cfa1a9a8b1e53a4f967db, reversing
changes made to dc46873e8db9abaf8ed2cef6423792195c0e5ff5.

Revert "Update cli version to 4.53.8"
This reverts commit 13884edfdd1407451daee36f03d5a02bfa701d11, reversing
changes made to 90cda3231d5f34951aa7b74535334180f980aef1.

Revert "Update cli version to 4.53.10"
This reverts commit f342d90e1ac9cf51ca9c5f748dc5933d528b0296, reversing
changes made to e04cf095c6833c98b2d5224e040ae999e279f516.

Revert "Update cli version to 4.53.11"
This reverts commit b44723a949fa5d18e1f40e1919c1da25ba75a64c, reversing
changes made to 029e13a07836392e5b48cf225a87fb3e83156a82.

Revert "Update cli version to 4.53.12"
This reverts commit 363a90ddd2f92c8f70eac94ffd40f3944de98def, reversing
changes made to e21d727233d406f593f06a14d6d5f97080322d7d.

Revert "Update cli version to 4.53.13"
This reverts commit 91c96c20893221c3f1519fbbc1fd9843d91fad96, reversing
changes made to b772f0ced84d733a5de99d89a7048b3485dd0a9f.

Revert "Update cli version to 4.53.15"
This reverts commit 91fcade0d7c161ed5d1d5319fc7027d59a2f3894, reversing
changes made to c58f2430af7289c8b6f608dbf413888840acd180.

Revert "Update cli version to alpha"
This reverts commit ed377ba09f7bd22ee49ced58b804880ba732c035, reversing
changes made to 793a9eab2f5db5c5e396b5337322bbacc03dc329.

Revert "Update cli version to ext-7.1"
This reverts commit 099325a42a8f6a0daaa4936efdc66db3cf37c12d, reversing
changes made to 3d39a276c915f5dfd37f05ad8318b0eaa2689557.

Revert "Update cli version to ext-7.1"
This reverts commit 20092b91a2edbc32ee72faade8d2ec79cd2be7d3, reversing
changes made to be51c7bdbcecb7073aac1d1a78664ccc82db52a9.

Revert "Update cli version to latest"
This reverts commit 45befb5de4d0c4b05b249390a1f5a2ce82b62bf5, reversing
changes made to 80373b2ed28ff2a657a8c7f5440fd91258655562.

Revert "Update cli version to ext-7.2"
This reverts commit 7b26594e5aace8abf382e7bd2927c7340a1ee20a, reversing
changes made to f95f0c56647dd7c22a4398be7b5042a6211d63d3.

Revert "Update cli version to ext-7.2"
This reverts commit 80df7dcd278bb0d8434f5be7056c5957cfa848b5, reversing
changes made to 5d7733044c5755a14613ccfe66b038c0136695c0.

Revert "Update cli version to ext-7.2"
This reverts commit 50e55d8a4011415477429fe14239b3fcde1668d9, reversing
changes made to 3d9ab59c380598ca562bef33a031d4df244669a2.

Revert "Update cli version to ext-7.2"
This reverts commit 6a28a40c1f0740a2b0527a582ba1982011bf75d2, reversing
changes made to 9b8d1eb774cbe95617839c4fecca56ac8f04060e.

Revert "Update plugins for new CLI"
This reverts commit 03b24e7a45bea13d810bbc13004b3d6a6020e38f, reversing
changes made to f399dc923c5837a23c19204893bac0f9bef9cc80.

Revert "Update cli version to ext-7.2"
This reverts commit b80d7501b50a3b3b8befad2c677ce10fd345e568, reversing
changes made to cca7bf43c87c8f251160705d9d934ee3d73aa068.

Revert "Fix plugins build"
This reverts commit a1b1e184505967f70ef1669985bf684d79b5ef2d, reversing
changes made to e35e4dbd012cdb0b00aae079e85cca904a3de0d8.

Revert "Update cli version to ext-7.2"
This reverts commit f16d3fd3f3d73947cb12414ef0f357f8161993a8, reversing
changes made to cc892794101b4eefebb44876829bf0344bf0c9a7.

Revert "Update cli version to ext-7.2"
This reverts commit b384b22f0c8ee2018d9d4ea23c58067ade643e71, reversing
changes made to e7bf50c25dd642067c7ad7a805bb8345bf3ebbc1.

Revert "Update cli version to ext-7.2"
This reverts commit 3bcef689c24fbfd19316944a8faac1843ecb21ae, reversing
changes made to 4e13eedd499f6db9a4f951df5a25ef76f52f01d8.

Revert "Update cli version to ext-7.2"
This reverts commit 1bd8419420848f4d1acfe33ba7e26e65cfa827b7, reversing
changes made to cde7c6eefb5b94bcffe577e52abf9f6439e44cf6.

Revert "Update cli version to ext-7.2"
This reverts commit 24a81d3ea873d05d18dcc46da21c3db6e7fe3f77, reversing
changes made to 660168756fc3cdac80cffc27acc5c9b9b9985eb2.

Revert "Update cli version to ext-7.2"
This reverts commit 851a87d4b103184b25403be4ec21a7b7a1c20d9e, reversing
changes made to 7734553fa5deecc8cc4385067cbfa651cc771121.

Revert "Update cli version to ext-7.2"
This reverts commit 6581b20496dfdce7762100c00665b131d3a64310, reversing
changes made to a699349f5d8621cad6b1802594790b9908a7ef00.

Revert "Update cli version to ext-7.2"
This reverts commit 0807e4e8604b68726b0a579c5ab4c634b0fd39f2, reversing
changes made to 9017dd6c55616005a21c4f3dc666191be66bd6f2.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503051308"
This reverts commit 84792fdc6e5415c09bd9bbcb65c957e8ea437f8d, reversing
changes made to 3d8df202656f999c560628fb76c5ba5548a830aa.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503051542"
This reverts commit ae5f9d2c365cf3ac4894e43dd49188554f4e2755, reversing
changes made to a7f6b4cb936f00657d938310ea1854c5c9307a11.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503052243"
This reverts commit 0ee8f18f22c579f4ae2dd13d76198094ff29b3c4, reversing
changes made to b06e4808d36354e55ab9536cb830f26a6b1b1ad8.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503060332"
This reverts commit 39f4313778c162541e9f16044827f0f8e0e78608, reversing
changes made to 21060d276b35c99f4f1fa6b1b2ca131e18b5982e.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503061202"
This reverts commit 4686c16012b131eb5e25e1254bbf4b318d35c0c0, reversing
changes made to 7ad420bff0b420e9e399e3f798b453574d9faee5.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503061415"
This reverts commit 3db23b1f3b3419a882320520aa2c478d89aa7a87, reversing
changes made to 68937d1b625c4e50ae26cbe95a0590fa88cc2736.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503062307"
This reverts commit eff25953e3a0ae884bba8e70bdbb96e2d5ba3bc9, reversing
changes made to 4a8b498f7eb5a912da4bb5c933a573deaeb7dc71.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503070057"
This reverts commit 2d5d0cbb30459b49ad45b0971a04bf77ca6caa56, reversing
changes made to 9993e4262d5665f74941415214e158a880cdfd18.

Revert "Add unarchive includer"
This reverts commit 67ade3b61c0e9a882b576f8528c123121499b4d6, reversing
changes made to de044589d3d930c3a1f91f8a9fd92cec67fd5c07.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503100646"
This reverts commit 882f491c04d7b77bb3d5b4b54446900b6b269b76, reversing
changes made to 41f034dd85afc9ac0d73eea5fc9c010e366c0622.

Revert "Update package-lock"
This reverts commit 3877e1e4a33921a7a6e21f6f85556c582f39c929, reversing
changes made to 7302ac1c4d831f6c841c906840f1937cf803d4fe.

Revert "Update cli to 4.56.0"
This reverts commit 92fefdebaeb9d5e6fa4645ad9cd48644dfa1c90f, reversing
changes made to 8f3498b2a434c3f72e0f5789ee69ae2aad71a05a.

Revert "Update cli version to 4.56.0"
This reverts commit eaff5355ac6f1d20f81455ac8c0c02b619b2f40c, reversing
changes made to 2b5a979679cd39db2fe1c9a67d955a7c2d3f4c94.

Revert "Update npm engine"
This reverts commit 4bf91457868571266fb7f91683365b4999e41768, reversing
changes made to 042d62ac24582d946f72265afaa712b8fffe449e.

Revert "Update cli version to 4.56.0"
This reverts commit 4241d06bc18af6ed7aa2e65d90a5ddfd4f5a3c18, reversing
changes made to 3a7b8e36af07f00f853679bcd63db3325e1a0369.

Revert "update yfm cli release"
This reverts commit 5088f55adb1fee05b32bf0a3dd887b03a9b5555b, reversing
changes made to 6d2b7d2c00c2d565781fc227fe6e0650e02c2b77.

Revert "Update cli version to 0.0.0-rc-ext-7.2-202503100646"
This reverts commit b19763cff8e5768a67c928cb3135c1914ee89704, reversing
changes made to 88d26126ed9cba414df861ba33e4043612c5dd29.

Revert "update package-lock.json"
This reverts commit b0c13e2417ec92ecede6086c3014478419bc1ec4, reversing
changes made to d95814ba1e3d5a5ea948aac6bc8abdfd48541bcb.

Revert "Update cli version to 0.0.0-rc-fix-lint-202503140002"
This reverts commit 8cb7070e7f44391dadbbbbb3a9c46f90d8a005a6, reversing
changes made to 6619da4eeb527c220c57b5cc4d34c60ec0881294.

Revert "Update cli version to 4.55.0"
This reverts commit 8d0e54a87719e039a1aa970521c87070ecf35390, reversing
changes made to 857b2315a667130184d707a94bc71645e160bb2b.

Revert "Update cli version to 4.54.2"
This reverts commit 134e2372e1c469bec84a5950500f23b0787f923d, reversing
changes made to e3075e2321694826d666bfa5f147a79de7b19183.

Revert "Update cli version to 4.50.0"
This reverts commit f341a0e45daa47027205f5f9cd654ad660547946, reversing
changes made to e0ab47e26bd986cb02b9247b0d0a56e8914c51e1.
commit_hash:becfde4a7d1b946ceed23f12cdb31002410c8797

robot-piglet committed:
Update contrib/libs/libunwind to 20.1.0
commit_hash:cb5c0561dc9a032f86c0e6d07d15517977d0ad47

ubyte committed:
annotate TJsonValue's methods with a lifetime bound attribute

Check that the strings and dictionaries owned by TJsonValue are not accessed after the destruction of the TJsonValue instance itself.
commit_hash:0ed6d84f28681d3f9d8c0fe7080bd7a5dd183919

alexv-smirnov committed:
Import libraries 250315-0050

robot-piglet committed:
Update contrib/libs/croaring to 4.3.0
commit_hash:9fd8100e998c2c83011096d6b8e59da7ca34a432

savnadya committed:
[kafka] YT-24188: Support CRC, fix Records format, add trace logging and fix cosmetics

* Changelog entry
Type: fix
Component: kafka-proxy

Fix Records format, calculate CRC
commit_hash:ced7dccd33df881dbbcb4d65b445454a69e7e19b

ivanmorozov333 committed:
header optimizers usage (#15675)

robot-piglet committed:
Update contrib/libs/cxxsupp/builtins to 20.1.0
commit_hash:dd1bc46d7dc6022db73a1089f5678a733f50180c

Commit:
Cosmetic yt/yt/library/formats/arrow_writer.h
commit_hash:d2124786f33227a4a95855919b31afd92e6ac91f

Nechda committed:
Enable include-angled-in-module-purview warnings
commit_hash:1f0af6e7d139ac5ec5aa16d8398b6128488b7c94

Commit:
Unify SetFrom and TrySetFrom signatures
commit_hash:a80d2b9c9c7ba35f4ee4f7bb645856989ed6e15c

Commit:
Cosmetics and refactoring in periodic executor
commit_hash:ead0cf707d59976550b0b4651cf6cd1ba4d4c678

Commit:
Cosmetics
commit_hash:6f864e4912a053569192d92bafb05e8b24e50ee8

yumkam committed:
fix large connector datasource tests with kqprun (#15780)

andrey-khropov committed:
Support switching between internal and external cython using USE_INTERNAL_CYTHON flag. The default behaviour will be to use external cython.. Fix #2810
commit_hash:4d8ef316fdb74d44e8cb22659188adaf3090a19a

alexv-smirnov committed:
Import libraries 250316-0050

maximyurchuk committed:
Cross version test tpch  (and other stress tools) (#15548)

Co-authored-by: Aidar Samerkhanov <aidarsamer@ydb.tech>
